Description: use local URL in an example in config/Safe.js
Author: Dmitry Shachnev <mitya57@debian.org>
Forwarded: not-needed
Last-Update: 2020-05-23

--- a/unpacked/config/Safe.js
+++ b/unpacked/config/Safe.js
@@ -9,7 +9,7 @@
  *  when you load MathJax.js, e.g.
  *  
  *  <script
- *    src="http://cdn.jsdelivr.net/npm/mathjax@2/MathJax.js?config=TeX-AMS_HTML,Safe">
+ *    src="file:///usr/share/javascript/mathjax/MathJax.js?config=TeX-AMS_HTML,Safe">
  *  </script>
  *
  *  ---------------------------------------------------------------------
